ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -203,18 +209,46 @@ public void onResume() {
   public void requestBannerAd(Context context, MediationBannerListener mediationBannerListener,
       Bundle serverParameters, AdSize adSize, MediationAdRequest mediationAdRequest,
       Bundle mediationExtras) {
+    // Create the adLayout wrapper with the requested ad size, as Vungle's ad uses MATCH_PARENT for
+    // its dimensions.
+    adLayout =
```

Comment:
minor: better to move back to VungleBannerAdapter.java  and hold only a reference here

Reply:
yes, good idea.